arm-resources/locks/2016-09-01/swagger/locks.json

```diff
@@ -325,6 +325,76 @@
             "description": "OK"
           }
         }
+      },
+      "get": {
```

Comment:
@vivsriaus @tianoms - Looks like these APIs were missing. @mcardosos from the Azure Go SDK team found this by looking at the MSDN REST API docs. Can you please review the changes?